Elevel/Elevel.Api/Controllers/AuditionController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Elevel.Application.Features.AuditionFeatures;
+using MediatR;
+
+namespace Elevel.Api.Controllers
+{
+
+    [Authorize, Route("[controller]/[action]")]
+    public class AuditionController : BaseApiController
+    {
+
+        [HttpPost]
+        public async Task<IActionResult> CreateAuditionAsync([FromBody] CreateAuditionCommand.Request request)
+        {
+            var res = await Mediator.Send(request);
+            return Ok(res);
+        }
+        [HttpPut("{id:Guid}")]
+        public async Task<IActionResult> UpdateAuditionAsync([FromRoute] Guid id, [FromBody] UpdateAuditionCommand.Request request)
```

Comment:
**Guid id** unnessesary because UpdateAuditionCommand.Request already contains this property.

Reply:
but it can be, is' double check for update.